### PR TITLE
 Note that Firefox and Chrome do not proxy localhost by default

### PR DESCRIPTION
**Pull request by kingthorin**
Re-implement https://github.com/zaproxy/zaproxy-website/pull/1120

Done via github.dev, I do agree with https://developercertificate.org/

### CODE REVIEW

**Comment by thc202** [comment]
They do proxy to localhost, they do not proxy localhost.

**Comment by kingthorin** [comment]
I thought I had reworded appropriately, could you be more specific about what I missed or the change you’d like.

File: site/content/faq/how-do-you-configure-zap-to-test-an-application-on-localhost.md [hunk 1]
```diff
@@ -16,4 +16,8 @@ Proxies](/docs/desktop/ui/dialogs/options/localproxy/) screen, remember to chang
 browser's proxy settings as well: [Configuring Proxies](/docs/desktop/start/proxies/).
 
 You also need to check that you have not configured your browser to ignore
-your configured proxy (ZAP) for localhost.
+the configured proxy (ZAP) for localhost. Note that Firefox (>= version 67)
+and Chrome (>= version 72) by default do not proxy traffic to loopback addresses
```
Comment (thc202) [inline site/content/faq/how-do-you-configure-zap-to-test-an-application-on-localhost.md:20]: Something like:
```suggestion
and Chrome (>= version 72) by default do not proxy loopback addresses' traffic
```

File: site/content/faq/how-do-you-configure-zap-to-test-an-application-on-localhost.md [hunk 2]
```diff
@@ -16,4 +16,8 @@ Proxies](/docs/desktop/ui/dialogs/options/localproxy/) screen, remember to chang
...
+(which includes localhost).
+[Configuring Proxies](/docs/desktop/start/proxies/) explains how to reconfigure
+them to proxy to loopback.
```
Comment (thc202) [inline site/content/faq/how-do-you-configure-zap-to-test-an-application-on-localhost.md:23]: This one would have to be tweaked too.

**Comment by kingthorin** [comment]
Thanks

File: site/content/faq/how-do-you-configure-zap-to-test-an-application-on-localhost.md (hunk 2 above)
Comment (kingthorin) [reply site/content/faq/how-do-you-configure-zap-to-test-an-application-on-localhost.md:23]:
```suggestion
them to loopback addresses' traffic.
```

**Comment by kingthorin** [comment]
Fixed up.

File: site/content/faq/how-do-you-configure-zap-to-test-an-application-on-localhost.md [hunk 3]
```diff
@@ -16,4 +16,8 @@ Proxies](/docs/desktop/ui/dialogs/options/localproxy/) screen, remember to chang
 browser's proxy settings as well: [Configuring Proxies](/docs/desktop/start/proxies/).
 
 You also need to check that you have not configured your browser to ignore
-your configured proxy (ZAP) for localhost.
+the configured proxy (ZAP) for localhost. Note that Firefox (>= version 67)
+and Chrome (>= version 72) by default do not proxy loopback addresses' traffic
+(which includes localhost).
+[Configuring Proxies](/docs/desktop/start/proxies/) explains how to reconfigure
+them to loopback addresses' traffic.
```
Comment (thc202) [inline site/content/faq/how-do-you-configure-zap-to-test-an-application-on-localhost.md:23]: `them to proxy …`

**Comment by kingthorin** [comment]
Got that one too :wink:

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you both!

**Review by psiinon — APPROVED** [review]

**Pull request merged**